src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -257,7 +257,17 @@ export default {
 				const messagesKeys = Object.keys(this.messages)
 				const lastMessageId = messagesKeys[messagesKeys.length - 1]
 
-				if (this.messages[lastMessageId].timestamp > lastMessageTimestamp) {
+				/**
+				 * Only use the last message as lastmessage when:
+				 * 1. It's newer than the conversations last message
+				 * 2. It's not a command reply
+				 * 3. It's not a temporary message starting with "/" which is a user posting a command
+				 */
+				if (this.messages[lastMessageId].timestamp > lastMessageTimestamp
+					&& (this.messages[lastMessageId].actorType !== 'bots'
+						|| this.messages[lastMessageId].actorId === 'changelog')
+					&& (!lastMessageId.startsWith('temp-')
```

Comment:
you didn't check if lastMessageId is a string here but did so further below ?

you can also convert an int to a string by doing String(lastMessageId).startsWith(...)
or if too dangerous, check for string with `typeof lastMessageId === 'string'`

Reply:
well it worked here, but failed on the other. Here I always got a string as id from the key list:


```
const object1 = {
  4: 'test'
};

console.log(Object.keys(object1));
// expected output: Array ["4"]
```
You can test it on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys

But I can still add it if you think it's needed

Reply:
fine to keep it as is if you're confident about the times